archive/r/ruby/file-io.rb

```diff
@@ -0,0 +1,23 @@
+def write_file
+    out = File.new("output.txt", "w")
+  
+    out << "This is a line written by a Ruby program\n"
+    out << "This line also"
+  
+    out.flush()
+    out.close()
+  end
```

Comment:
Idk if the mobile UI is having issues here, but it looks like the indentation is messed up from here on down.